client/src/components/Query/Statement.jsx

```diff
@@ -327,6 +328,24 @@ class Statement extends React.Component {
     }
   }
 
+   handleNewQuery() {
+     const { draft , display } = this.state;
+     const key = uuidv1();
+     const instructions = cloneDeep(DEFAULT_INSTRUCTIONS).instructions
+
+     const draftQuery = {   instructions ,
+                            "key": key}
+     draft.push(draftQuery)
+
+     display.push(this.props.display)
```

Comment:
@claudia1296 Apply cloneDeep; or changing the display for that new query might also change the default display setting :)